apkid/rules/apk/packers.yara

```diff
@@ -41,6 +41,24 @@ rule appguard : packer
     is_apk and ($stub and $encrypted_dex)
 }
 
+rule appguard_new : packer
```

Comment:
I know version info isn't always available, but are there any clues in this case?
Otherwise looks great.

Reply:
I have no idea Caleb, sorry.